_includes/head.html

```diff
@@ -41,6 +41,8 @@
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
     ga('create', 'UA-6700447-1', 'auto');
+    ga('require', 'linker');
+    ga('linker:autoLink', ['www.defacto.nl', 'jobs.defacto.nl']);
```

Comment:
Shouldn't www.defacto.nl be removed from the autolink? 

```javascript
ga('require', 'linker');
ga('linker:autoLink', ['example-2.com'] );
```

also  don't we need this `allowLinker` ?
```javascript
 ga('create', 'UA-XXXXXXX-Y', 'auto', {'allowLinker': true});
```